observability/otel/otel-rn.adoc

```diff
@@ -12,9 +12,89 @@ You can use the {OTELName} xref:otel-forwarding-telemetry-data.adoc#otel-forward
 
 include::snippets/distr-tracing-and-otel-disclaimer-about-docs-for-supported-features-only.adoc[]
 
-// git rebase marker 1
+[id="otel_3-4_{context}"]
+== Release notes for {OTELName} 3.4
 
-// git rebase marker 2
+The {OTELName} 3.4 is provided through the link:https://catalog.redhat.com/software/containers/rhosdt/opentelemetry-operator-bundle/615618406feffc5384e84400/history[{OTELOperator} <x>.<y>.<z>].
+
+The {OTELName} 3.4 is based on the open source link:https://opentelemetry.io/docs/collector/[OpenTelemetry] release 0.<y>.0.
+
+[id="otel_3-4_cves_{context}"]
+=== CVEs
+
+This release fixes the following CVEs:
+
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
+* link:https://access.redhat.com/security/cve/CVE-2024-????[CVE-2024-????]
```

Comment:
@andreasgerstmayr, how many of the DT CVEs can I copy to here.

Reply:
None from my side, all the CVEs I added are for Jaeger UI.